### PR TITLE
Refactored environment file.

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: src/chainbridgeConfig.ts [hunk 1]
```diff
@@ -61,14 +61,14 @@ export const chainbridgeConfig: ChainbridgeConfig = {
...
-      rpcUrl: process.env.REACT_APP_POLYGON_RPC_URL as string,
+      rpcUrl: "",
       type: "Ethereum",
       nativeTokenSymbol: "MATIC",
       tokens: [
         {
           address: process.env.REACT_APP_POLYGON_ERC20_TOKEN_ADDRESS as string,
-          name: process.env.REACT_APP_POLYGON_ERC20_NAME as string,
-          symbol: process.env.REACT_APP_POLYGON_ERC20_SYMBOL as string,
+          name: "CERENETWORK",
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: The value should be like in PROD.

File: src/chainbridgeConfig.ts [hunk 2]
```diff
@@ -108,14 +108,14 @@ export const chainbridgeConfig: ChainbridgeConfig = {
...
-      rpcUrl: process.env.REACT_APP_ETHEREUM_RPC_URL as string,
+      rpcUrl: "",
       type: "Ethereum",
       nativeTokenSymbol: "ETH",
       tokens: [
         {
           address: process.env.REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS as string,
-          name: process.env.REACT_APP_ETHEREUM_ERC20_NAME as string,
-          symbol: process.env.REACT_APP_ETHEREUM_ERC20_SYMBOL as string,
+          name: "CERENETWORK",
```
Comment (AndreiNavoichyk) [inline src/chainbridgeConfig.ts]: Do not forget DRY. Move value to constant.

File: src/chainbridgeConfig.ts (hunk 1 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Upated.

File: src/chainbridgeConfig.ts (hunk 2 above)
Comment (karthikeyan-cere) [reply src/chainbridgeConfig.ts]: Updated.

File: .env.dev [hunk 3]
```diff
@@ -17,21 +17,15 @@ REACT_APP_ETHEREUM_NETWORK_ID="5"
 REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Goerli"
```
Comment (AndreiNavoichyk) [inline .env.dev:16]: What is `REACT_APP_SUPPORT_SERVICE`?

File: .env.dev [hunk 4]
```diff
@@ -17,21 +17,15 @@ REACT_APP_ETHEREUM_NETWORK_ID="5"
 REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Goerli"
 REACT_APP_ETHEREUM_BRIDGE_ADDRESS="0xa806cA3bD88F790744462cBC34c40EDd5b8dc2Dd"
```
Comment (AndreiNavoichyk) [inline .env.dev:17]: What is `REACT_APP_SUPPORT_URL`

File: .env.stage [hunk 5]
```diff
@@ -17,20 +17,14 @@ REACT_APP_ETHEREUM_NETWORK_ID="5"
 REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Goerli"
```
Comment (AndreiNavoichyk) [inline .env.stage:16]: Align structure with `.env.dev`

File: .env.prod [hunk 6]
```diff
@@ -17,10 +17,7 @@ REACT_APP_ETHEREUM_NETWORK_ID="1"
 REACT_APP_ETHEREUM_NETWORK_NAME="Ethereum Mainnet"
```
Comment (AndreiNavoichyk) [inline .env.prod:17]: Align structure with `.env.stage`

File: .env.dev (hunk 3 above)
Comment (karthikeyan-cere) [reply .env.dev:16]: Updated value

File: .env.dev (hunk 4 above)
Comment (karthikeyan-cere) [reply .env.dev:17]: Updated telegram url

File: .env.stage (hunk 5 above)
Comment (karthikeyan-cere) [reply .env.stage:16]: Updated.

File: .env.prod (hunk 6 above)
Comment (karthikeyan-cere) [reply .env.prod:17]: updated

Comment (AndreiNavoichyk) [inline .env.prod:17]: Why did you skip `*SUPPORT*` for PROD?

File: .env.dev [hunk 7]
```diff
@@ -1,9 +1,8 @@
 PORT=3000
 HTTPS=true
 
-REACT_APP_BLOCKNATIVE_DAPP_ID=
```
Comment (AndreiNavoichyk) [inline .env.dev:4]: What value will be for BLOCKNATIVE by default?

File: .env.prod (hunk 6 above)
Comment (karthikeyan-cere) [reply .env.prod:17]: I will update all the prod values in a separate PR. Once dev and stage is tested.

File: .env.dev (hunk 7 above)
Comment (karthikeyan-cere) [reply .env.dev:4]: Empty.

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**